buku

```diff
@@ -5276,10 +5278,12 @@ def main():
     """Main."""
     global ID_STR, ID_DB_STR, MUTE_STR, URL_STR, DESC_STR, DESC_WRAP, TAG_STR, TAG_WRAP, PROMPTMSG
     # readline should not be loaded when buku is used as a library
+    import readline
     try:
-        import readline
+        import colorama
+        colorama.just_fix_windows_console()  # noop on non-Windows systems
     except ImportError:
-        import pyreadline3 as readline  # type: ignore
```

Comment:
`pyreadline3` (…and incidentally `pyreadline` as well) actually [exports a `readline` module](https://github.com/pyreadline3/pyreadline3/blob/master/readline.py), so this fallback export is both redundant and useless.